### DISCUSSION TITLE
JSON Examples

### DISCUSSION THREAD

**Discussion by Khaleel** [nolimits4web/swiper#4580] [Q&A]
Hi Guys
I have a CMS that gives me a JSON feed for slides. However I cannot find examples of how to use SwiperJS with JSON feed to loop through slides. I am very surprised that the main documentation does not even list how to iterate through an array of JSON and prepend slides?

**Comment by nolimits4web** [comment]
How is it related specifically to Swiper? It is a common JavaScript/HTML on how to loop through JSON and make/generate HTML from it ;)
